diff --git a/node_modules/@uiw/react-md-editor/esm/components/DragBar/index.js b/node_modules/@uiw/react-md-editor/esm/components/DragBar/index.js
index e2f1444..5bb5c91 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/DragBar/index.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/DragBar/index.js
@@ -1,5 +1,4 @@
 import React, { useEffect, useMemo, useRef } from 'react';
-import "./index.css";
 import { jsx as _jsx } from "react/jsx-runtime";
 var DragBar = props => {
   var {
diff --git a/node_modules/@uiw/react-md-editor/esm/components/TextArea/Textarea.js b/node_modules/@uiw/react-md-editor/esm/components/TextArea/Textarea.js
index 4f43ca9..891a9cc 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/TextArea/Textarea.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/TextArea/Textarea.js
@@ -7,7 +7,6 @@ import { EditorContext } from "../../Context.js";
 import { TextAreaCommandOrchestrator } from "../../commands/index.js";
 import handleKeyDown from "./handleKeyDown.js";
 import shortcuts from "./shortcuts.js";
-import "./index.css";
 import { jsx as _jsx } from "react/jsx-runtime";
 export default function Textarea(props) {
   var {
diff --git a/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.js b/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.js
index c82d522..69abc4e 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.js
@@ -7,7 +7,6 @@ import shortcuts from "./shortcuts.js";
 import Markdown from "./Markdown.js";
 import Textarea from "./Textarea.js";
 import { TextAreaCommandOrchestrator } from "../../commands/index.js";
-import "./index.css";
 import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 export default function TextArea(props) {
   var _ref = props || {},
diff --git a/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.nohighlight.js b/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.nohighlight.js
index 5d7a8c7..8be7648 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.nohighlight.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/TextArea/index.nohighlight.js
@@ -6,7 +6,6 @@ import { EditorContext } from "../../Context.js";
 import shortcuts from "./shortcuts.js";
 import Textarea from "./Textarea.js";
 import { TextAreaCommandOrchestrator } from "../../commands/index.js";
-import "./index.css";
 import { jsx as _jsx } from "react/jsx-runtime";
 export default function TextArea(props) {
   var _ref = props || {},
diff --git a/node_modules/@uiw/react-md-editor/esm/components/Toolbar/Child.js b/node_modules/@uiw/react-md-editor/esm/components/Toolbar/Child.js
index b50d314..2bd5491 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/Toolbar/Child.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/Toolbar/Child.js
@@ -1,6 +1,5 @@
 import _extends from "@babel/runtime/helpers/extends";
 import React, { useContext, useMemo } from 'react';
-import "./Child.css";
 import Toolbar from "./index.js";
 import { EditorContext } from "../../Context.js";
 import { jsx as _jsx } from "react/jsx-runtime";
diff --git a/node_modules/@uiw/react-md-editor/esm/components/Toolbar/index.js b/node_modules/@uiw/react-md-editor/esm/components/Toolbar/index.js
index 2abe1f3..67576e2 100644
--- a/node_modules/@uiw/react-md-editor/esm/components/Toolbar/index.js
+++ b/node_modules/@uiw/react-md-editor/esm/components/Toolbar/index.js
@@ -2,7 +2,6 @@ import _extends from "@babel/runtime/helpers/extends";
 import React, { Fragment, useContext, useEffect, useRef } from 'react';
 import { EditorContext } from "../../Context.js";
 import Child from "./Child.js";
-import "./index.css";
 import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 export function ToolbarItems(props) {
   var {
diff --git a/node_modules/@uiw/react-md-editor/esm/index.js b/node_modules/@uiw/react-md-editor/esm/index.js
index 743eb1c..c457ed8 100644
--- a/node_modules/@uiw/react-md-editor/esm/index.js
+++ b/node_modules/@uiw/react-md-editor/esm/index.js
@@ -1,7 +1,6 @@
 import MDEditor from "./Editor.js";
 import * as commands from "./commands/index.js";
 import * as MarkdownUtil from "./utils/markdownUtils.js";
-import "./index.css";
 export { headingExecute } from "./commands/title.js";
 export * from "./commands/index.js";
 export * from "./commands/group.js";
diff --git a/node_modules/@uiw/react-md-editor/esm/index.nohighlight.js b/node_modules/@uiw/react-md-editor/esm/index.nohighlight.js
index f1e0f2b..5fcc5d6 100644
--- a/node_modules/@uiw/react-md-editor/esm/index.nohighlight.js
+++ b/node_modules/@uiw/react-md-editor/esm/index.nohighlight.js
@@ -1,7 +1,6 @@
 import MDEditor from './Editor.nohighlight';
 import * as commands from "./commands/index.js";
 import * as MarkdownUtil from "./utils/markdownUtils.js";
-import "./index.css";
 export { headingExecute } from "./commands/title.js";
 export * from "./commands/index.js";
 export * from "./commands/group.js";
