diff --git a/node_modules/react-comments-section/dist/index.js b/node_modules/react-comments-section/dist/index.js
index 7e8502f..10a594f 100644
--- a/node_modules/react-comments-section/dist/index.js
+++ b/node_modules/react-comments-section/dist/index.js
@@ -3,14 +3,11 @@ function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'defau
 var React = require('react');
 var React__default = _interopDefault(React);
 var Picker = _interopDefault(require('emoji-picker-react'));
-require('react-draft-wysiwyg/dist/react-draft-wysiwyg.css');
 var draftJs = require('draft-js');
 var reactDraftWysiwyg = require('react-draft-wysiwyg');
 var draftToHtml = _interopDefault(require('draftjs-to-html'));
 var htmlToDraft = _interopDefault(require('html-to-draftjs'));
 var reactMenu = require('@szhsin/react-menu');
-require('@szhsin/react-menu/dist/core.css');
-require('react-responsive-modal/styles.css');
 var reactResponsiveModal = require('react-responsive-modal');
 
 var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
diff --git a/node_modules/react-comments-section/dist/index.modern.js b/node_modules/react-comments-section/dist/index.modern.js
index 98d033a..587b141 100644
--- a/node_modules/react-comments-section/dist/index.modern.js
+++ b/node_modules/react-comments-section/dist/index.modern.js
@@ -1,13 +1,10 @@
 import React__default, { createContext, useState, useEffect, useRef, useContext, createElement } from 'react';
 import Picker from 'emoji-picker-react';
-import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
 import { EditorState, ContentState, convertToRaw } from 'draft-js';
 import { Editor } from 'react-draft-wysiwyg';
 import draftToHtml from 'draftjs-to-html';
 import htmlToDraft from 'html-to-draftjs';
 import { Menu, MenuItem } from '@szhsin/react-menu';
-import '@szhsin/react-menu/dist/core.css';
-import 'react-responsive-modal/styles.css';
 import { Modal } from 'react-responsive-modal';
 
 var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
@@ -17673,8 +17670,7 @@ var AdvancedInput = function AdvancedInput(_ref) {
   }, "Post")))));
 };
 
-var _require = require('uuid'),
-  uuidv4 = _require.v4;
+import { v4 as uuidv4 } from 'uuid';
var InputField = function InputField(_ref) {
  var formStyle = _ref.formStyle,
    comId = _ref.comId,
diff --git a/node_modules/@solana/wallet-adapter-react-ui/styles.css b/node_modules/@solana/wallet-adapter-react-ui/styles.css
index 4cb7b2f..a0e19c1 100644
--- a/node_modules/@solana/wallet-adapter-react-ui/styles.css
+++ b/node_modules/@solana/wallet-adapter-react-ui/styles.css
@@ -1,4 +1,3 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap');
 
 .wallet-adapter-button {
     background-color: transparent;
     border: none;
diff --git a/node_modules/@tronweb3/tronwallet-adapter-react-ui/style.css b/node_modules/@tronweb3/tronwallet-adapter-react-ui/style.css
index 6e4d2a0..9f7e6b8 100644
--- a/node_modules/@tronweb3/tronwallet-adapter-react-ui/style.css
+++ b/node_modules/@tronweb3/tronwallet-adapter-react-ui/style.css
@@ -1,4 +1,3 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap');
 
 /** button **/
 .adapter-react-button {
